packages/sn-filter-pane/package.json

```diff
@@ -45,13 +45,14 @@
     "@babel/preset-typescript": "^7.18.6",
     "@emotion/react": "11.10.4",
     "@emotion/styled": "11.10.4",
-    "@nebula.js/cli": "^3.0.4",
-    "@nebula.js/cli-build": "^3.0.4",
-    "@nebula.js/cli-sense": "^3.0.4",
-    "@nebula.js/cli-serve": "^3.0.4",
+    "@nebula.js/cli": "^3.1.0-alpha.2",
+    "@nebula.js/cli-build": "^3.1.0-alpha.2",
+    "@nebula.js/cli-sense": "^3.1.0-alpha.2",
+    "@nebula.js/cli-serve": "^3.1.0-alpha.2",
```

Comment:
We should probably replace this in listbox too (find and replace in whole repo) to make them aligned.